app/controllers/users_controller.rb

```diff
@@ -30,7 +30,15 @@ def update
     redirect_to users_url
   end
 
-  def none; end
+  def none
+    # rubocop:disable Lint/PercentStringArray
+    override_content_security_policy_directives(
+      script_src: %w[*.hsforms.net *.hsforms.com 'self'],
+      img_src: %w[*.hsforms.com *.hsforms.net 'self'],
+      style_src: %w['self' 'unsafe-inline']
```

Comment:
the single quotes are required: https://github.com/twitter/secure_headers/blob/master/docs/per_action_configuration.md